python/lsst/analysis/ap/plotImageSubtractionCutouts.py

```diff
@@ -50,6 +49,44 @@
 from . import apdb
 
 
+class ButlerCache:
+    """Global class to handle butler queries, to allow lru_cache and
+    multiprocessing.pool to work together.
+
+    If we redo this all to work with BPS or other parallelized systems, or get
+    good butler-side caching, we could remove this lru_cache system.
+    """
+    def set(self, butler, config):
+        """Call this to store a Butler and pex.Config instance before using the
+        global class instance.
```

Comment:
Should you have full doc strings here?

Reply:
I've added a `Parameters` section, but it's not really any more informative than that sentence was.